native/src/modules/app/components/App.js

```diff
@@ -15,29 +14,30 @@ import type { DataContainer } from '../../endpoint/DataContainer'
 import NavigatorContainer from '../containers/NavigatorContainer'
 import { SafeAreaProvider } from 'react-native-safe-area-context'
 import StaticServerProvider from '../../static-server/containers/StaticServerProvider'
+import { NavigationContainer } from '@react-navigation/native'
 
-class App extends React.Component<{||}> {
-  dataContainer: DataContainer = new DefaultDataContainer()
-  store: Store<StateType, StoreActionType> = createReduxStore(this.dataContainer)
+const dataContainer: DataContainer = new DefaultDataContainer()
+const store: Store<StateType, StoreActionType> = createReduxStore(dataContainer)
 
-  render () {
-    return (
-      <Provider store={this.store}>
-        <StaticServerProvider>
-          <I18nProviderContainer>
-            <SafeAreaProvider>
-              <>
-                <StatusBarContainer />
-                <IOSSafeAreaView>
+const App = () => {
+  return (
+    <Provider store={store}>
+      <StaticServerProvider>
+        <I18nProviderContainer>
+          <SafeAreaProvider>
+            <>
+              <StatusBarContainer />
+              <IOSSafeAreaView>
+                <NavigationContainer>
                   <NavigatorContainer />
-                </IOSSafeAreaView>
-              </>
-            </SafeAreaProvider>
-          </I18nProviderContainer>
-        </StaticServerProvider>
-      </Provider>
-    )
-  }
+                </NavigationContainer>
```

Comment:
One is a `NavigatioNContainer` from the library which is definitely needed. The other one is a self defined `NavigatoRContainer` which is a wrapper (with connect) around our Navigator defining all routes. I don't mind changing the naming though if you think there is a better name.

Reply:
ah okay. i missed `Navigator` and `Navigation`
Maybe we could call the `NavigatorContainer` something like `RoutesContainer` if that makes sense? Otherwise it is difficult to understand whats going on here

Reply:
@maxammann what do you think?